Marlin/Configuration.h

```diff
@@ -530,27 +530,30 @@
 #define Z_MIN_PROBE_ENDSTOP_INVERTING false // set to true to invert the logic of the probe.
 
 /**
- * Specify Stepper Driver types
- * The options are used to determine driver pulse timings as well as more advanced functionality.
- * Stepper timing options can be overridden in Configuration_adv.h
+ * Stepper Drivers
+ *
+ * These settings allow Marlin to tune stepper driver timing and enable advanced options for
+ * stepper drivers that support them. You may also override timing options in Configuration_adv.h.
+ *
+ * A4988 is assumed for unspecified drivers.
```

Comment:
@thinkyhead That's not quite right. The assumption is made only if no drivers are specified.

Reply:
Semantics. The individual driver is assumed to be A4988. The fact that the code compiles based on the slowest driver is not a detail exposed to users.